src/vs/workbench/contrib/notebook/browser/view/cellParts/cellOutput.ts

```diff
@@ -329,9 +329,9 @@ class CellOutputElement extends Disposable {
 		const isFirstCellOutput = NOTEBOOK_CELL_IS_FIRST_OUTPUT.bindTo(menuContextKeyService);
 		const cellOutputMimetype = NOTEBOOK_CELL_OUTPUT_MIMETYPE.bindTo(menuContextKeyService);
 		isFirstCellOutput.set(index === 0);
-		if (mimeTypes[index]) {
-			cellOutputMimetype.set(currentMimeType.mimeType);
-		}
+
+		cellOutputMimetype.set(currentMimeType.mimeType);
```

Comment:
no need to only conditionally set this that I can think of, and the previous condition did not make sense